pkg/controller/load_configuration.go

```diff
@@ -0,0 +1,96 @@
+package controller
+
+import (
+	"context"
+	"os"
+	"strings"
+
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+// LoadFromSecret retrieves an operator secret
```

Comment:
Can we add more details to the function description? What exactly this function does. What are the given params, etc. In what case an error is returned and what cases are ignored (no secret name is specified or exists, etc).
This function now lives in `common` so it now requires a better documentation :)

Reply:
done here: 26869a4875e8152516ff11b763471d23defe1670